pkg/plugin/v3/webhook.go

```diff
@@ -96,6 +100,19 @@ func (p *createWebhookPlugin) Validate() error {
 			" --programmatic-validation and --conversion to be true", p.commandName)
 	}
 
+	if p.conversion {
+		if (p.spoke == "" && p.hub != "") || (p.hub == "" && p.spoke != "") {
+			return fmt.Errorf("Hub and Spoke version need to be specified when using conversion webhooks")
+		}
+		if p.spoke == "" || p.hub == "" {
+			fmt.Println("Hub and Spoke version can be specified in the command to scaffold Convertible and Hub interfaces")
+			return nil
+		}
+		if p.spoke == p.hub {
+			return fmt.Errorf("Cannot have the same hub and spoke version")
+		}
+	}
+
```

Comment:
I think we need to start to cover the code with tests. WDYT about add a webhook_test here to check these scenarios?